pkgs/servers/monitoring/uptime-kuma/fix-database-permissions.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/server/server.js b/server/server.js
+index 0c9a45e6..cec31c7c 100644
+--- a/server/server.js
++++ b/server/server.js
+@@ -1583,6 +1583,7 @@ async function initDatabase(testMode = false) {
+     if (! fs.existsSync(Database.path)) {
+         log.info("server", "Copying Database");
+         fs.copyFileSync(Database.templatePath, Database.path);
++        fs.chmodSync(Database.path, 0o755);
```

Comment:
Sure this shpuld be 755?

Reply:
Well, it is how the file are set when uptime-kuma is packaged via docker so I didn't go so much further. I believe setting it to something more restrictive like 644 would work here but I didn't know if it was my place to suggest a change like that.

Reply:
We cannot compare Docker to NixOS. NixOS is a shared system with many other users and services while in Docker kuma is with itself.

Reply:
I agree, I was just trying not to introduce breaking changes, as I wish to merge the patch upstream. Will change perms to 640 for the patch and we'll see how it goes upstream